### PR TITLE
Todomvc es6 updates

### PR DESCRIPTION
**Pull request by flashdesignory**
- renamed todoMVC workload
- added proper package.json
- updated readme file
- renamed "js" to "src"
- add S3 formatting
- add build script (this moves all necessary files to a dist folder for the tests)
- add dev server (for local preview)
- pin node version (same as s3)
- fixed toggle all
- removed node_modules folder from github (since we have a package-lock files)

No refactoring of logic or styling updates have been implemented.

### CODE REVIEW

File: resources/todomvc/vanilla-examples/javascript-es6/dist/template.js [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-    '&': '&amp',
-    '<': '&lt',
-    '>': '&gt',
-    '"': '&quot',
-    '\'': '&#x27',
-    '`': '&#x60'
+    "&": "&amp",
+    "<": "&lt",
+    ">": "&gt",
+    '"': "&quot",
```
Comment (rniwa) [inline resources/todomvc/vanilla-examples/javascript-es6/dist/template.js:8]: It seems inconsistent to use '' here. Why not just "\""?

**Comment by rniwa** [comment]
Do we have before/after values?

**Comment by flashdesignory** [comment]
> Do we have before/after values?

Here's a gist of the average of 10 runs:
https://gist.github.com/flashdesignory/42ac2fbabb18b4c527672509f157442a

File: resources/todomvc/vanilla-examples/javascript-es6/dist/template.js (hunk 1 above)
Comment (flashdesignory) [reply resources/todomvc/vanilla-examples/javascript-es6/dist/template.js:8]: I ran it through prettier and eslint 🤷 
It's consistent with the other todomvc apps, but I can manually adjust it if needed.

**Review by rniwa — APPROVED** [review]

**Review by bgrins — APPROVED** [review]
Looks good to me too, feel free to land

**Pull request merged**